src/components/Card.vue

```diff
@@ -18,21 +21,48 @@
         </el-tooltip>
       </div>
       <div v-if="showCardDetails" class="details">
-        <p>
+        <p v-if="!data.hideType">
           <b>{{ data.type }}</b>
         </p>
-        <el-tooltip :content="data.title" placement="top">
-          <p class="title">
-            {{ data.title }}
-          </p>
-        </el-tooltip>
-        <el-button @click.prevent="openLinkInNewTab"> View {{ data.type }}</el-button>
+        <el-popover ref="galleryPopover" :content="data.title" placement="top" trigger="hover" popper-class="gallery-popper" />
+        <p v-if="!data.hideTitle" class="title" v-popover:galleryPopover>
+          {{ data.title }}
+        </p>
+        <el-button class="button" @click.prevent="cardClicked"> View {{ data.type }}</el-button>
       </div>
     </div>
   </el-card>
 </template>
 
 <script>
+// import { SvgIcon } from '@abi-software/svg-sprite'
+import axios from 'axios'
+import Vue from 'vue'
+import { Button, Card, Popover } from 'element-ui'
+Vue.use(Button)
+Vue.use(Card)
+Vue.use(Popover)
```

Comment:
Should this not all be in the install function? Is it needed after importing for this component?

Reply:
This is needed to support the current mapvuer setup at present. I will look into optimise these setups once we get all the styles in sync with sparc-app using the design components.

Reply:
Vue.use are required in this way. The import itself does not put those components into vue.

Reply:
It is not common to use vue npm packages this way